### ISSUE TITLE
Arrows not Auto Binding / ATTACHING on Shapes

### ISSUE THREAD

**Issue by orvpagadua** [excalidraw/excalidraw#7997]
Arrows used to work fine but after the update its not longer attaching to shapes which drives me insane. Please ADD the feature back.

**Comment by dwelle** [comment]
Hi. Arrow binding is still supported, but we scaled down the auto-binding. From now on you need to bind by dragging the arrow end points.

See https://twitter.com/excalidraw/status/1786055557645824177 for more.

**Comment by orvpagadua** [comment]
can we make auto binding the default setting?

**Comment by dwelle** [comment]
No, we've just moved away from it as per user feedback. But we may consider supporting the old behavior through a config.

**Comment by gg-mmill** [comment]
Hello, 
what about binding to custom shapes (created with line or pen tool ) ? I'm not sure this ever worked, but at least now it doesn't seem to work, is this intentional ?

**Comment by dwelle** [comment]
This never worked, but we'll add binding to lines at some point. Pen tool likely as well.
